### PR TITLE
Merge feature/virtualbackground onto master

### PR DESCRIPTION
**Pull request by gabriel-jt**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### JIRA link(s):

- [VIDEO-6414](https://issues.corp.twilio.com/browse/VIDEO-6414)

### Description

Merge the virtual background feature onto master

### Before review
* [x] Updated CHANGELOG.md if necessary
* [ ] Added unit tests if necessary
* [ ] Updated affected documentation
* [x] Verified locally with `npm test`
* [x] Manually sanity tested running locally
* [ ] Included screenshot as PR comment (if needed)
* [x] Ready for review

### Before merge
* [ ] Got one or more +1s
* [ ] Re-tested if necessary

### CODE REVIEW

**Review by charliesantos — APPROVED** [review]
Let's get @timmydoza 's +1 before merging.

File: src/components/MenuBar/MenuBar.tsx [hunk 1]
```diff
@@ -90,12 +90,12 @@ export default function MenuBar() {
...
               {!isSharingScreen && !isMobile && <ToggleScreenShareButton disabled={isReconnecting} />}
               {process.env.REACT_APP_DISABLE_TWILIO_CONVERSATIONS !== 'true' && <ToggleChatButton />}
+              <Menu />
             </Grid>
           </Grid>
           <Hidden smDown>
             <Grid style={{ flex: 1 }}>
-              <Grid container justifyContent="flex-end">
-                <Menu />
+              <Grid container justify="flex-end">
```
Comment (timmydoza) [inline src/components/MenuBar/MenuBar.tsx:98]: This should be `justifyContent="flex-end"`. There's a warning in the console about it (but only when when browser width is reduced to show the mobile styles). 

image

File: package.json [hunk 2]
```diff
@@ -16,10 +17,11 @@
     "@types/linkify-it": "^3.0.0",
     "@types/lodash.throttle": "^4.1.6",
     "@types/node": "^12.12.26",
-    "@types/react": "^16.9.19",
+    "@types/react": "17.0.11",
```
Comment (timmydoza) [inline package.json:20]: Let's keep React at `^16.9.19`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+## 0.5.0 (August 6, 2021)
+
+### New Feature
+
+- This release adds a virtual background feature. This feature allows users to blur their background or apply a background image from the background selection window. The virtual backgrounds are applied to the video tracks using the [Twilio Video Processors SDK](https://www.twilio.com/docs/video/video-processors). For more information, please view this [blog post](https://www.twilio.com/blog/introducing-virtual-backgrounds-browser-based-video-applications).
```
Comment (timmydoza) [inline CHANGELOG.md:5]: Let's add a link to the PR at the end of the description:
```suggestion
- This release adds a virtual background feature. This feature allows users to blur their background or apply a background image from the background selection window. The virtual backgrounds are applied to the video tracks using the [Twilio Video Processors SDK](https://www.twilio.com/docs/video/video-processors). For more information, please view this [blog post](https://www.twilio.com/blog/introducing-virtual-backgrounds-browser-based-video-applications). [#574](https://github.com/twilio/twilio-video-app-react/pull/574)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,9 @@
...
+
```
Comment (timmydoza) [inline CHANGELOG.md:6]: Also here let's add information about the package upgrades that happened in #568.

```
### Dependency Upgrades
-`@material-ui/core` has been updated from 4.9.1. to 4.12.3. [#568](https://github.com/twilio/twilio-video-app-react/pull/568)
-`@material-ui/icons` has been updated from 4.9.1. to 4.11.12. [#568](https://github.com/twilio/twilio-video-app-react/pull/568)
-`@twilio/conversations` has been updated from 1.1.0. to 1.2.3. [#568](https://github.com/twilio/twilio-video-app-react/pull/568)
```

**Review by timmydoza — CHANGES_REQUESTED** [review]
Nearly there! Just some quick changes needed...

**Comment by timmydoza** [comment]
Also, when I run `npm install`, there is a change made to `package-lock.json`. Let's commit that change.

**Review by charliesantos — APPROVED** [review]

**Review by timmydoza — APPROVED** [review]

**Pull request merged**